MsmqReturnToSourceQueue/ErrorManager.cs

```diff
@@ -89,7 +98,7 @@ public void ReturnMessageToSourceQueue(string messageId)
 
                         if (string.IsNullOrEmpty(originalId) || messageId != originalId)
                         {
-                            if (messageCount%ProgressInterval == 0)
+                            if (messageCount % ProgressInterval == 0)
```

Comment:
please keep refactors in separate commits. This helps if things need to be reverted

Reply:
I didn't think it was a big deal for what is probably a one-off modification.

Reply:
Dammit I fixed it and it keeps sneaking in every time I save. IMO this should not hold up a merge.